src/nunit-gui/Views/MainForm.cs

```diff
@@ -189,5 +189,11 @@ protected override void OnDragDrop(DragEventArgs drgevent)
             if (files != null)
                 DragDropFiles?.Invoke(files);
         }
+
+        private void TabControlSelectedIndexChanged(object sender, System.EventArgs e)
+        {
+            if (tabControl1.SelectedIndex == 1)
```

Comment:
I know you said not to merge, but can this be changed to check that the selected tab is the xml tab and not based off of the index?

Reply:
Done. Please review 😄

Reply:
I'm all for it. Merge when ready!